packages/ui/components/combobox/src/LionCombobox.js

```diff
@@ -1090,6 +1178,6 @@ export class LionCombobox extends LocalizeMixin(OverlayMixin(LionListbox)) {
 
   clear() {
     super.clear();
-    this._inputNode.value = '';
+    this.value = '';
```

Comment:
When FormatMixin is applied, it would delegate this to _inputNode. Does it do in this case as well?

Reply:
Yes, this is delegated as can be seen in the tests.

Reply:
Nvm, see the getter now 👍